packages/fuels-signers/src/wallet.rs

```diff
@@ -312,32 +313,16 @@ impl Wallet {
         .flatten()
         .collect();
 
-        // This script loads:
-        //  - a pointer to the contract id,
-        //  - the actual amount
-        //  - a pointer to the asset id
-        // into the registers 0X10, 0x11, 0x12
-        // and calls the TR instruction
```

Comment:
Are these comments still relevant? Why remove them?

Reply:
You're right, I forgot to return them.